lib/js/styles/Deprecated/Colors.stories.mdx

```diff
@@ -10,7 +10,6 @@ Color names generator can be found here: https://www.color-blindness.com/color-n
 
 ## Theme colors
 <ColorPalette>
-  <ColorItem title="Primary" colors={[THEME_COLORS.PRIMARY]} />
```

Comment:
is it intentionally removed? I'd keep it here until we fully migrate to new colors

Reply:
No, I'm reverting this change.